src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -497,6 +501,7 @@ class NexusPublishPluginTests {
     }
 
     @Test
+    @Disabled("Fails on my Fedora...")
```

Comment:
Ups, I have to do some conditional checking to exclude only my system as it's hard to keep it locally and do not commit accidentally...